radio/src/gui/128x64/model_setup.cpp

```diff
@@ -234,7 +234,11 @@ void onModelAntennaSwitchConfirm(const char * result)
 #define EXTERNAL_ANTENNA_ROW
 #endif
 
-#if defined(PCBX7) || defined(PCBX9LITE)
+#if defined(PCBX7ACCESS)
```

Comment:
#if defined(PCBX7ACCESS) || defined(PCBXLITES)

it will avoid to duplicate the code